src/app/api/jhipster-mocks/repositories/service.ts

```diff
@@ -0,0 +1,72 @@
+import { Repository } from '@prisma/client';
+
+import { db } from '@/app/api/jhipster-mocks/db';
+
+export type RepositoryFormatted<U extends Partial<Repository> = Repository> =
+  ReturnType<typeof formatRepositoryFromDb<U>>;
+
+export const formatRepositoryFromDb = <U extends Partial<Repository>>(
+  repository?: U | null
+) => {
+  return !!repository ? repository : undefined;
+};
+
+export const prepareRepositoryForDb = <
+  U extends Partial<RepositoryFormatted & { password?: string }>
+>(
+  user: U
+) => {
+  // Format fields for database
+  return {
+    ...user,
+  };
+};
```

Comment:
Be careful, we are not using user in this file.

Reply:
I think those functions are useless for this entity. `User` contains sensitive data, but not `Repository`